packages/server/lib/project-base.ts

```diff
@@ -308,46 +300,6 @@ export class ProjectBase<TServer extends Server> extends EE {
     options.onError(err)
   }
 
-  async initializeSpecsAndDevServer (updatedConfig: Cfg): Promise<{
```

Comment:
Nice, I love deleting code from project-base - I think we should be mostly getting rid of this entire thing, eventually.